src/pages/examples.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Composition, Box } from 'atomic-layout'
+
+import Layout from '../components/layout'
+import { Grid } from '../components/Grid'
+import { ExampleItem } from '../components/exampleItem'
+import { TextLead } from '../components/TextLead'
+import { ReactComponent as GraphQLLogo } from '../images/graphql-logo.svg'
+import { ReactComponent as JavaScriptIcon } from '../images/logos/javascript-2.svg'
+import { ReactComponent as StorybookIcon } from '../images/logos/storybook.svg'
+import { Heading } from '../components/Heading'
+
+const ExampleList: React.FC = ({ children }) => {
+  return (
+    <Composition
+      gap={64}
+      templateCols="repeat(1, 1fr)"
```

Comment:
`repeat(1, 1fr)` is the same as `1fr`, is the same as not giving this prop at all :) You can remove `templateCols` prop.